### PR TITLE
Update pt-BR translation

### PR DESCRIPTION
**Pull request by gildaswise**
I was going to add the pt-BR translation but couldn't find the time to do it these past few weeks. I started reviewing the current version of the portuguese translation and there's stuff that's straight up wrong and not considering the proper word order in portuguese. Some text seems like it was generated by a GPT or another AI tool, or even Google Translate, instead of a proper translation. I'll try to finish this as soon as I can.

The worst offender so far was this:

```
#: src/view/com/util/forms/PostDropdownBtn.tsx:236
msgid "Appeal content warning"
msgstr "Aviso de conteúdo de apelação"
```

It translates back to "Warning about appealing content", the correct translation would be "Contestar aviso de conteúdo". There's many others but this is definitely not a good translation and I believe a review by another language speaker should be required before merging such translation PRs.

Another offender:

```
#: src/view/com/profile/ProfileHeader.tsx:369
msgid "Unmute Account"
msgstr "Não silenciar conta"
```

Translates back to "Don't silence account", correct translation would be "Desmutar conta"

### CODE REVIEW

**Comment by gildaswise** [comment]
I believe I'm done, I used POEditor to edit this and it formatted the .po file in a weird way, please let me know if you would like to see this reverted.

**Comment by maisondasilva** [comment]
Hi @gildaswise, how are you? You need to maintain the same file structure, because when new variables are processed this will break the translation because when processing several tsx are changed!
I believe you saw that as soon as you sent the translation, new tsx were added and changed!
You commented on the translation "Desmutar conta" but the correct translation would be "Desativar silencio da conta"
GTP Chat was not used but Crowdin.
Regarding words with some incorrect translations, they are still being adjusted with the mobile and web application based on the number of characters that can be displayed, if possible, correct them in my open pull.
For example, the translation "Comentários" broke the application and in the new version I sent I kept "Feedback" so that when developers adjust the number of characters it is easier to resolve!

https://github.com/bluesky-social/social-app/pull/2528

If you could review this link I would appreciate it, because as you changed the file structure the language may break on the next version!
You can edit directly from Github in this case!

Thanks

**Comment by gildaswise** [comment]
Hey, @maisondasilva 

Did you use the Crowdin AI feature? That's unfortunately the same as using ChatGPT to translate. Bluesky recently had an issue with a third party translation company also just using AI tools and calling it a day, so it's really unfair to do the same again here

I'm not sure "length of characters" is a valid excuse for having translated text that doesn't match what it should mean; the first impression that I had by reviewing the first translation and now your new PR is that you didn't proofread the translation, as you can see in the fact that you keep using "Publicação" in some random places instead of "Post" which I believe is the better usage since Threads also doesn't translate that (same with "story" on Instagram)

There's a few places where plural is being used instead of possession for "their" translations too

The translation done in this PR was entirely by hand by checking what other apps were using; I opted for "desmutar" instead of "desativar silêncio" due to the length of that but maybe "dessilenciar" is the best option between the available ones

**Comment by maisondasilva** [comment]
@gildaswise An observation that the initial translation was to Portuguese (PT) after it was sent it was changed to Brazilian Portuguese (PT-BR), and as a result it was redone more than once because once I sent the file it was updated several times and with that the IDs changed and as it was the end of the year, some people revised it (few actually), but the idea was to send the translation to receive new variations and then do the necessary polishing, the initial idea being Portuguese. As there were few people to review and I did it alone, it is normal that some things may be wrong.
I believe it's a good idea for them to accept my pull and then you make a new pull and request a review so that other people can help. Since you have changed the structure of the translation, it is important that you maintain the same line quality of the translation.
https://github.com/bluesky-social/social-app/pull/2287
Your contributions are welcome, as this is a community and it is important that translations are reviewed by more than one person!
But the translation is still being polished, as several translations are being changed.
As for Discord, it is a voice over IP and textual communication application where it is not a social network.

First pull
https://github.com/bluesky-social/social-app/pull/2407

add tag for BR Portuguese
https://github.com/bluesky-social/social-app/pull/2407/commits/c9e069950d09bd4bc7a5ecca3994f67dd89c4d8a

**Comment by maisondasilva** [comment]
@gildaswise The translation has just received new variables, if you wish, you can cancel your pull and I will cancel my pull as it will overwrite yours, and you send a new pull maintaining the structure of the original file and with the new variables.

I used Crowdin as an Editor not IA because I have a repository with this translation there! I even suggested that programmers use Crowdin and an integration with GIthub, so people could help there and the files would be updated automatically.

**Comment by gildaswise** [comment]
> An observation that the initial translation was to Portuguese (PT) after it was sent it was changed to Brazilian Portuguese (PT-BR)

This makes things even worse and explains some changes that I had to make. I unfortunately wasn't following the PRs for the pt-BR translation and couldn't give my input on this back then.

> Since you have changed the structure of the translation, it is important that you maintain the same line quality of the translation.

Not sure what you mean by this, the quality of the current translation is not to be followed since it has grammar and syntax issues. If it's just the formatting of the file, I'll fix this tomorrow. 

> As for Discord, it is a voice over IP and textual communication application where it is not a social network.

Discord is also a social network, and they're using the same word for muting profiles, channels and servers, it's not the same as muting audio in this case.

> The translation has just received new variables, if you wish, you can cancel your pull and I will cancel my pull as it will overwrite yours, and you send a new pull maintaining the structure of the original file and with the new variables.

I'll update this PR with the file formatted properly tomorrow.

**Comment by maisondasilva** [comment]
@gildaswise Just one detail you told me "The worst offender so far was this:"

Another offender:

#: src/view/com/profile/ProfileHeader.tsx:369
msgid "Unmute Account"
msgstr "Não silenciar conta"

But in your translation you used the word
msgid "Dessilenciar conta"

In short, the word "Dessilenciar" this word is not recognized by the official dictionary of the Portuguese language. However, I understand that it would mean similar "Unmute"

In conclusion, the word "desilencing" does not currently exist in the Portuguese language. And you used it in several places in the translation!
I believe that using a word that is not yet recognized is a much worse offender!

I believe that this PR he must reviewed by other speakers of the language to ensure that the translation is more correct, the intention here is to help and unfortunately you arrived with ignorance when you arrived!

**Comment by maisondasilva** [comment]
@ansh

**Review by gleydson — COMMENTED** [review]
Excellent work here, amigo! Here are some typos I found and suggestions.

**Comment by gildaswise** [comment]
@gleydson 

> Excellent work here, amigo! Here are some typos I found and suggestions.

Thank you for the review! Sorry about those.

**Comment by maisondasilva** [comment]
@gildaswise I don't know which editor you used but this made 10 translation variables stay on the same line, I left it in the comments above!

Example
image

Could you use the structure and add back the tsx and comments that you removed from the translations, so that the file maintains the same number of lines as the original file?

Example on line 4120, but I noticed that there are other lines like this!

image

**Comment by gildaswise** [comment]
I used the POEditor, I can fix this later today

**Comment by maisondasilva** [comment]
Are you referring to the En file or the pt-BR?
Because the line `#: src/view/screens/Moderation.tsx: 205`, which is not a comment, I don't see in your file at the end!

**Comment by gildaswise** [comment]
I've reordered the file already, there's only two in the end of the file that don't have a file/line linked. The string at Moderation.tsx 205 changed to 206, it's on line 320.

**Comment by maisondasilva** [comment]
@gildaswise In the En file there is still 205, but if you notice the translation comments are different, that's why I'm informing you! I believe that now it may be clearer what I am talking about!

Image below are from the En file

image

image

**Comment by gildaswise** [comment]
Got it, not sure where POEdit placed those. That's done now, though from what I understood, these strings are not used anymore. Their content isn't there in the source file and line.

**Comment by pfrazee** [comment]
Sounds like we're ready to merge?

**Comment by gildaswise** [comment]
@pfrazee I'm finishing up a last review through a friend that works with translations, it'll be up soon and then it'll be ready!

**Comment by faeriarum** [comment]
@pfrazee translation has been reviewed and should be ready to go!

**Comment by pfrazee** [comment]
Excellent! Thank you so much everybody!

**Comment by gildaswise** [comment]
@pfrazee Updated with 1.66 strings

**Review by pfrazee — APPROVED** [review]
Thanks everyone!
